e2e/cypress/e2e/superadmin.cy.ts

```diff
@@ -1,25 +1,93 @@
-// TODO better selectors
-
 import {
-  clickSuperadmin, insertCredentials, loginAsAdmin, logout, logoutAdmin, resetBackendData, visitLoginPage
+  addWorkspaceAdmin, clickSuperadmin, insertCredentials, loginAdmin, logout, logoutAdmin, resetBackendData
 } from './utils';
 
-describe('Superadmin', () => {
-  beforeEach(resetBackendData);
-  beforeEach(logout);
-  beforeEach(() => loginAsAdmin());
-  beforeEach(clickSuperadmin);
+// ########################## Credentials #################################################
```

Comment:
Sowas ist eigentlich zu unterlassen. Man kanns in bestimmten Fällen machen, hier scheint mir das nicht nötig.

Reply:
Hab ich wieder in den alten Zustand gebracht.